docs/explanation/configs.rst

```diff
@@ -32,6 +34,44 @@ value resolution is as follows:
 3. config file from ``/etc/xdg``
 4. config file from ``$SNAP_DATA``
 
+Check the configuration
+=======================
+
+The values resolution order and the fact that configurations can be stored in
+so many different places may bring confusion when running Checkbox.
+
+Fortunately, the ``check-config`` command will list:
+
+- all the configuration files being used
+- for each section, the configured parameters being used
+- the origin of each of these customized parameters
+- an overall status report
+
+For example:
+
+.. code-block:: none
+
+    $ checkbox-cli check-config
+
+    Configuration files:
+     - /var/snap/checkbox/2799/checkbox.conf
+       [config]
+         config_filename=checkbox.conf      (Default)
+       (...)
+       [test plan]
+         filter=*                           (Default)
+         forced=False                       (Default)
+         unit=                              (Default)
+       [test selection]
+         exclude=                           (Default)
+         forced=False                       (Default)
+       (...)
+       [environment]
+         STRESS_BOOT_ITERATIONS=100         From config file: /var/snap/checkbox/2799/checkbox.conf
+       (...)
+       [manifest]
+    No problems with config(s) found!
```

Comment:
What happens if a problem occurs? Could you add a sample output? (If users need to check their config file, a link to the configuration reference doc would also be helpful)

Reply:
Very good idea! I've added all that in https://github.com/canonical/checkbox/pull/671/commits/ba5f43bfa29e51cbb1cbe8628a576dd3c086557d